packages/webview_flutter/example/lib/main.dart

```diff
@@ -3,11 +3,26 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert';
 import 'package:flutter/material.dart';
 import 'package:webview_flutter/webview_flutter.dart';
 
 void main() => runApp(MaterialApp(home: WebViewExample()));
 
+const String kNavigationExamplePage = '''
+<!DOCTYPE html><html>
+<body>
```

Comment:
the <html> is optional but the <title> is not. :-)

Reply:
done.

Getting @Hixie  to review my HTML:

<img src="https://media.giphy.com/media/UgM7H8OEmf4mQ/giphy.gif" width="480" height="269"  />

😄